uportal-war/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/entry.jsp

```diff
@@ -273,6 +273,7 @@
         </a>
     </li>
     <li class="divider"></li>
+    {% if (canFavorite) { %}
     <li>
         <spring:message code="add.this.portlet.to.my.favorite" text="Add this Portlet to My Favorites" var="atptmfTitle"/>
```

Comment:
This message already exists in [Messages.properties#L47](https://github.com/Jasig/uPortal/blob/master/uportal-war/src/main/resources/properties/i18n/Messages.properties#L47)